<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>List</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTreeWidget" name="memo_tree">
         <column>
          <property name="text">
           <string>Memo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Mod Date</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>새 항목</string>
          </property>
          <property name="text">
           <string>1912</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>새 항목</string>
          </property>
          <property name="text">
           <string>123</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>새 항목</string>
          </property>
          <property name="text">
           <string>23</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>새 항목</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>새 항목</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>새 항목</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Content</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QTextEdit" name="memo_content"/>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="memo_title"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
